src/org/labkey/targetedms/SkylineDocImporter.java

```diff
@@ -410,7 +433,6 @@ private ReplicateInfo insertReplicates(TargetedMSRun run, SkylineDocumentParser
 
             if (folderType == TargetedMSService.FolderType.QC)
             {
-                // In QC folders insert a replicate only if at least one of the associated sample files will be inserted
```

Comment:
I see what you mean about the current behavior - I retract my previous defense of this comment. :)